charts/bpa/values.yaml

```diff
@@ -66,6 +66,13 @@ bpa:
     i18n:
       locale: en
       fallbackLocale: en
+    runtimeVariables:
+      # -- Enables runtime variables which are injected into the frontend code on container startup (only if at least one runtime variable value has been set). These function e.g. as feature toggles for specific frontend changes. NOTE: The container startup time is impacted by using these.
+      enabled: false
```

Comment:
This is not needed as you can check the property directly in the configmap

Reply:
Done